cockroachcloud/backups-page.md

```diff
@@ -1,28 +1,42 @@
 ---
 title: Restore Data from a Backup
-summary: Restore a table or database from a backup in CockroachDB Cloud.
+summary: Restore data from a backup in CockroachDB Cloud.
 toc: true
 docs_area: manage
 ---
 
-This page describes the **Backups** page and how to restore your data.
+<div class="filters clearfix">
+    <button class="filter-button page-level" data-scope="serverless"><strong>{{ site.data.products.serverless }}</strong></button>
+    <button class="filter-button page-level" data-scope="dedicated"><strong>{{ site.data.products.dedicated }}</strong></button>
+</div>
 
-Cockroach Labs runs [full backups](../{{site.versions["stable"]}}/take-full-and-incremental-backups.html#full-backups) daily and [incremental backups](../{{site.versions["stable"]}}/take-full-and-incremental-backups.html#incremental-backups) hourly for every {{ site.data.products.db }} cluster. The full backups are retained for 30 days, while incremental backups are retained for 7 days.
+
+This page describes the **Backups** page and how to restore your data.
 
 The backups that Cockroach Labs runs for you can be viewed on the [Backups page](#backups-page).
 
+<section class="filter-content" markdown="1" data-scope="serverless">
+Cockroach Labs runs [full backups](../{{site.versions["stable"]}}/take-full-and-incremental-backups.html#full-backups) hourly for every {{ site.data.products.serverless }} cluster. The full backups are retained for 30 days.
```

Comment:
For Serverless, should this line be "Cockroach Labs runs [full cluster backups] hourly...." cc @livlobo